src/openms_gui/source/VISUAL/DIALOGS/ToolsDialog.cpp

```diff
@@ -446,13 +446,23 @@ namespace OpenMS
     // If we can't get any valid strings show an error.
     String extension = FileTypes::typeToName(FileTypes::UNKNOWN);
     auto validStrings = vis_param_.getValidStrings(output_combo_->currentText().toStdString()); 
-    if (!validStrings.empty())
+    // If we have only one valid output type use that
+    if (validStrings.size() == 1)
     {
       extension = validStrings[0];
+      // Remove the leading .*
+      extension = extension.suffix(extension.size() - 2);
     }
-    else
+    // Otherwise the type is unknown
+    else 
     {
-      QMessageBox::critical(this, "Error", QString("Error determining output type from tool. Tool is not compatible with TOPPView"));
+      // If we have no valid types, produce an error
+      if (validStrings.empty())
+        {
+          QMessageBox::critical(this, "Error", QString("Error determining output type from tool. Tool is not compatible with TOPPView"));
+        }
+          // If we have multiple valid output types, we don't know what the file actually contains, so use UNKNOWN
+
```

Comment:
there is something missing here?!

Reply:
we could also (as a precaution) emit a warning message when we detect multiple extensions and print something like
```
Multiple possible output aure supported by the tool. You probably need to set the output type in the parameters (usually `out_type`).
```

Reply:
> ?!

Nope. We set extension to UNKNOWN above, so we don't need to do anything here. Just putting a comment in so that the logic flow is clear.

Reply:
> we could also (as a precaution) emit a warning message when we detect multiple extensions and print something like
> 
> ```
> Multiple possible output aure supported by the tool. You probably need to set the output type in the parameters (usually `out_type`).
> ```
> 
> to the log window

I'm merging this now since it's actively a problem for the work Timo is doing. I'll open another PR with the suggestion.

Reply:
see https://github.com/OpenMS/OpenMS/pull/7785/